site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -211,7 +211,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `gameservers.podPreserveUnknownFields`                   | Disable [field pruning][pruning] and schema validation on the Pod template for a [GameServer][gameserver] definition | `false`                |
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                             | `8000`                 |
 
-{{% feature publishVersion="1.24.0" %}}
+{{% feature publishVersion="1.25.0" %}}
```

Comment:
There are more `feature` shortcodes to be cleaned up, for example in metrics.md

A good search is to do a global search for `% feature` across all .md files.